README.md

```diff
@@ -259,6 +266,21 @@ export class CdkTestStack extends Stack {
       ],
       true
     );
+    NagSuppressions.addResourceSuppressions(
+      thirdUser,
+      [
+        {
+          id: 'AwsSolutions-IAM5',
+          reason: 'Suppress AwsSolutions-IAM5 on the SQS resource.',
+          appliesTo: [
+            {
+              regex: /^Resource::arn:aws:sqs:(.*):\*$/g.toString(),
```

Comment:
```suggestion
              regex: '/^Resource::arn:aws:sqs:(.*):\\*$/g',
```
The documentation gets best effort [translated to other languages on ConstructHub ](https://constructs.dev/packages/cdk-nag/v/2.7.5?lang=python#suppressing-a-rule). This is more language agnostic